src/cctag/cuda/frame_07d_vote_if.cu

```diff
@@ -64,140 +59,35 @@ bool Frame::applyVoteIf( )
 
     NumVotersIsGreaterEqual select_op( _voters.dev );
 
+#if 0
```

Comment:
maybe we could define a variable to activate this piece of code from cmake and report this bug in a `known issues` section in some of the doc file (README or BUILD)?
```cmake
option(CCTAG_NO_THRUST_COPY_IF "Do not use thrust::copy_if(), circumvent bugs in cuda 7 +  GTX 980 and GTX 980 Ti GTX 1080  " OFF)

# then is the if(CCTAG_WITH_CUDA) part
if(CCTAG_NO_THRUST_COPY_IF)
target_compile_definitions(CCTag  PRIVATE -DCCTAG_NO_THRUST_COPY_IF)
endif()
                         
# also add the entry in the config.hpp.in
```

Reply:
Good idea, just pushed.